ci_framework/roles/artifacts/tasks/main.yml

```diff
@@ -14,6 +14,11 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Verify if "success" flag exists after successful tests execution
```

Comment:
This seems to be dirty way of passing status. However I don't know about any better way of passing statuses between `pre`, `run` and `post` steps.

Reply:
that also work outside of zuul itself, so it's probably not too wrong in the end :)